app/views/posts/_post.html.erb

```diff
@@ -29,6 +29,12 @@
           <% end %>
         </div>
 
+        <div class="item__details-item">
```

Comment:
I don't think I like this to be honest, I think I'd rather keep the post overview as clean as possible.

Reply:
Fair enough. I'll let you determine where you want to display the players.